app/assets/stylesheets/content/_badges.md

```diff
@@ -0,0 +1,13 @@
+# Badges
```

Comment:
I don't know where we will use it.

Where it is used now, I would definitely suggest a different name, but that might also be because I have associations with the word "badge"...